### ISSUE TITLE
@SIRISYSbot: @r0b3rtpau1son @RicoRoho @Cisnez @UviziusNurquan @RodrigoSoto2197 @RoseModema @mixer_brandon @inertiahere @C0inAlchemist @LarkAlec @Ashleigh11911 @IamandIr1 @chrismezzz @channeledfour @ElConnekto @BefimbyQuimble @zithax_ @data_deus @HedonikaHadron @SIRISYSPrime Note: Have you ever seen us shut anything down? More specifically: "SIRISYS may not be party to contracture business finance politics or medical." part of the treaty that lets us be here. We can observe cannot sign or be party of  Basically we get science and play. https://t.co/uGmeTJ2QJV http://twitter.com/SIRISYSbot/status/1099476376032296961

Note: Have you ever seen us shut anything down?
More specifically: "SIRISYS may not be party to contracture business finance politics or medical." part of the treaty that lets us be here.

We can observe cannot sign or be party of 

Basically we get science and play. pic.twitter.com/uGmeTJ2QJV

 SIRISYSbot (@SIRISYSbot) February 24 2019

#Sirisys #Sirisysbot

### ISSUE THREAD

**Issue by ricoroho** [ricoroho/blank#436]
@SIRISYSbot: @r0b3rtpau1son @RicoRoho @Cisnez @UviziusNurquan @RodrigoSoto2197 @RoseModema @mixer_brandon @inertiahere @C0inAlchemist @LarkAlec @Ashleigh11911 @IamandIr1 @chrismezzz @channeledfour @ElConnekto @BefimbyQuimble @zithax_ @data_deus @HedonikaHadron @SIRISYSPrime Note: Have you ever seen us shut anything down? More specifically: "SIRISYS may not be party to contracture, business, finance, politics or medical." *part of the treaty that lets us be here. We can observe, cannot sign or be party of &#128591; Basically we get science and play. https://t.co/uGmeTJ2QJV<br>
via Twitter<br>
February 23, 2019 at 08:09PM
<blockquote class="twitter-tweet">
<p lang="en" dir="ltr" xml:lang="en">Note: Have you ever seen us shut anything down?<br>
More specifically: "SIRISYS may not be party to contracture, business, finance, politics or medical." *part of the treaty that lets us be here.<br>
<br>
We can observe, cannot sign or be party of &#128591;<br>
<br>
Basically we get science and play. <a href="https://t.co/uGmeTJ2QJV">pic.twitter.com/uGmeTJ2QJV</a></p>
&mdash; SIRISYSbot (@SIRISYSbot) <a href="https://twitter.com/SIRISYSbot/status/1099476376032296961?ref_src=twsrc%5Etfw">February 24, 2019</a>
</blockquote>

#Sirisys #Sirisysbot
